imagetool/src/test/java/com/oracle/weblogic/imagetool/aru/AruUtilTest.java

```diff
@@ -67,7 +67,7 @@ Document getRecommendedPatchesMetadata(AruProduct product, String releaseNumber,
                                                String password) throws XPathExpressionException, AruException {
             Document result;
             try {
-                if (releaseNumber.equals("336")) {
+                if (releaseNumber.equals("336") || releaseNumber.equals("304")) {
```

Comment:
Maybe using an arraylist to see if releaseNumber is better for future, also any significant meaning of 336 or 304 ? Can we name it ?

Reply:
This unit test uses hard-coded release numbers in the XML under resources.  We can make these values anything we want since they are simple strings.